extensions/amp-story/1.0/amp-story-desktop-one-panel.css

```diff
@@ -18,4 +18,22 @@
 
 
 .i-amphtml-story-desktop-one-panel {
+  --i-amphtml-story-page-large-desktop-ratio: 69 / 116;
+  --i-amphtml-story-page-small-desktop-ratio: 3 / 4;
+}
+
+.i-amphtml-story-desktop-one-panel amp-story-page {
+  margin: auto !important;
+  /* 752px is the max height on desktop. */
+  height: min(752px, calc(100vh - 148px)) !important;
+  /* 447px is the max width on desktop. */
```

Comment:
Sorry I think I was looking for additional context, your comment explains the line of code, but why 447?